### PR TITLE
Nv 295 delete notification template

### PR DESCRIPTION
**Pull request by ninjadev101**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
#504

### CODE REVIEW

File: apps/api/src/app/notification-template/usecases/delete-notification-template/delete-notification-template.usecase.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    await this.notificationTemplateRepository.update(
+      {
+        _id: command.templateId,
+        _organizationId: command.organizationId,
+      },
+      {
+        $set: {
+          isDeleted: true,
+        },
```
Comment (djabarovgeorge) [inline apps/api/src/app/notification-template/usecases/delete-notification-template/delete-notification-template.usecase.ts:27]: Currently, we use 'mongoose-delete' on several repositories, it helps us to manage the variable such as isDeleted, deletedAt, and deletedBy. I think we could use it here as well, what do you think? 
For further examples, you can look into the integration repository and its use there.

**Review by djabarovgeorge — COMMENTED** [review]
We have not had a chance to talk yet but I heard you are doing a wonderful job from the other contributors. The PR looks really good I left one comment for your review.

File: apps/api/src/app/notification-template/usecases/delete-notification-template/delete-notification-template.usecase.ts (hunk 1 above)
Comment (ninjadev101) [reply apps/api/src/app/notification-template/usecases/delete-notification-template/delete-notification-template.usecase.ts]: I had a discussion with @scopsy yesterday and I started copy-pasting that... Smart Coder :)

**Comment by scopsy** [comment]
This is great @ninjadev101!

**Pull request merged**